The shipped build.xml is completely bogus in trying to bring in maven.
We just bring in ant-core, codemodel and istack-commons-runtime-1.1 ourselves,
and it compiles perfectly.

Signed-off-by: Robin H. Johnson <robbat2@gentoo.org>

diff -Nuar istack-commons-20090620.orig/buildtools/build.xml istack-commons-20090620/buildtools/build.xml
--- istack-commons-20090620.orig/buildtools/build.xml	2009-03-30 19:28:53.000000000 -0700
+++ istack-commons-20090620/buildtools/build.xml	2009-06-20 22:15:09.194598087 -0700
@@ -1,6 +1,8 @@
 <?xml version="1.0"?>
 <project name="istack-commons-buildtools" basedir="." default="dist" xmlns:artifact="urn:maven-artifact-ant">
+	<property name="name" value="istack-commons-buildtools" />
     <import file="../build-common.xml"/>
+	<!--
     <property name="maven.ant.file" value="maven-ant-tasks-2.0.9.jar"/>
     <property name="src" value="src"/>
     <property name="lib.dir" value="lib"/>
@@ -79,5 +81,6 @@
             <fileset dir="${src}"/>
         </jar>
     </target>
+	-->
 
 </project>
